notificationservice.cpp

```diff
@@ -46,6 +46,7 @@ public slots:
                     else if(mReader.name() == "ai") appIcon = mReader.readElementText();
                     else if(mReader.name() == "su") summary = mReader.readElementText();
                     else if(mReader.name() == "bo") body = mReader.readElementText();
+                    else if(mReader.name() == "vb") vibrate = mReader.readElementText();
```

Comment:
I don't think we should use "vb" and "vibrate". It's not super important yet but ultimately the goal of ngfd is to unify all sorts of feedbacks (including audio notifications for example) into a single API. Having another abbreviation might be slightly more future-proof.